backend/build.gradle.kts

```diff
@@ -13,6 +13,7 @@ plugins {
     // Apply the Kotlin JVM plugin to add support for Kotlin.
     id("org.jetbrains.kotlin.jvm") version "1.8.0"
     id("org.jlleitschuh.gradle.ktlint") version "11.0.0"
+    id("org.jetbrains.kotlin.android") version "1.8.0" apply false
```

Comment:
What does that do? :D

Reply:
I tried resolving my issue with `Class 'kotlin.Unit' was compiled with an incompatible version of Kotlin. The binary version of its metadata is 1.8.0, expected version is 1.6.0.`. But this only helped in some places, so I'll remove it again.